projects/packages/jetpack-mu-wpcom/src/features/wpcom-whats-new/wpcom-whats-new.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * WPCom Whats New support for WordPress.com sites.
+ *
+ * @package automattic/jetpack-mu-plugins
+ */
+
+define( 'MU_WPCOM_WHATS_NEW', true );
+
+/**
+ * Check if the WPCom Whats New should be loaded.
+ *
+ * @return bool
+ */
+function should_load_wpcom_whats_new_app() {
+	// Only load on the WPcom platform.
+	if ( ! class_exists( 'Automattic\Jetpack\Status\Host' ) ) {
+		return false;
+	}
+
+	global $pagenow;
+	$allowed_pages = array(
+		'post.php',
+		'post-new.php',
+		'site-editor.php',
+	);
+	return isset( $pagenow ) && in_array( $pagenow, $allowed_pages, true );
+}
+
+/**
+ * Enqueue block editor assets.
+ */
+function wpcom_whats_new_enqueue_script_and_style() {
+	if ( ! should_load_wpcom_whats_new_app() ) {
+		return;
+	}
+
+	$whats_new_handle = 'wpcom-whats-new';
+	$version          = gmdate( 'Ymd' );
```

Comment:
Does this mean that the file will only cached per-day, and would require a new download each day?

Reply:
Yes.